src/Microsoft.ML.KMeansClustering/KMeansCatalog.cs

```diff
@@ -16,20 +17,45 @@ public static class KMeansClusteringExtensions
         /// <summary>
         /// Train a KMeans++ clustering algorithm.
         /// </summary>
-        /// <param name="ctx">The regression context trainer object.</param>
-        /// <param name="features">The features, or independent variables.</param>
+        /// <param name="ctx">The clustering context trainer object.</param>
+        /// <param name="featureColumn">The features, or independent variables.</param>
```

Comment:
Maybe `featureColumnName`? In ML.NET, we mix the meanings of column name, column index, and column itself. I feel our naming can be more specific and therefore less ambiguous.

Reply:
I will leave it to featureColumn, because we have had a loooonnnggg discussion about it, and a PR to standartize those.

[https://github.com/dotnet/machinelearning/pull/1524#discussion_r231231728](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231231728)

---
In reply to: [248383325](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248383325) [](ancestors = 248383325)